src/api/TencentMeeting.ts

```diff
@@ -35,7 +35,8 @@ const requestWithBody = (body: string, options: {
     const req = (options.protocol === 'https:' ? https : http)
       .request(options, callback);
     req.on('error', (e: Error) => {
-      reject(e);
+      console.error('Request error:', e);
+      reject(new Error('腾讯会议错误'));
```

Comment:
I don't see how this change could result in better error message in the frontend. have you tested it?

I would change the following:

in errors.ts, add:

```
export const generalError = (message: string) =>
  new TRPCError({ code: 'INTERNAL_SERVER_ERROR', message });
```

then here:

```
throw generalError('腾讯会议后台错误，请重试');
```

please also make sure you test it out.